generator/templates/override.py

```diff
@@ -7,6 +7,13 @@ class Instance:
       - a list of strings as first parameters
       - the parameters given as the constructor parameters (must be strings)
     """
+
+    # logging related private fields
+    _vsnprintf = None
+    _max_log_message_size = 4096
+    _logger = None
+    _log_handler = None
+
```

Comment:
You are totally right, I will remove these.
Instead of `getattr`, I can also initialize the _logger in the __new__ method (or override the __init__).
Which one do you prefer ? getattr or __new__/__init__ ?

Reply:
Nevermind, the `__new__` already has its own logic, no need to add more complexity in it.